src/cli/cli.cpp

```diff
@@ -550,6 +570,28 @@ template <> otError Interpreter::Process<Cmd("ba")>(Arg aArgs[])
     return error;
 }
 
+void Interpreter::OutputBorderAgentCounters(const otBorderAgentCounters &aCounters)
+{
+    OutputFormat("ePSKc: activations %lu ", ToUlong(aCounters.mEpskcActivations));
+    OutputFormat("api_deactivations %lu ", ToUlong(aCounters.mEpskcDeactivationClears));
+    OutputFormat("timeout_deactivations %lu ", ToUlong(aCounters.mEpskcDeactivationTimeouts));
+    OutputFormat("max_attempt_deactivations %lu ", ToUlong(aCounters.mEpskcDeactivationMaxAttempts));
+    OutputLine("disconnect_deactivations %lu ", ToUlong(aCounters.mEpskcDeactivationDisconnects));
+    OutputFormat("ePSKc: invalid_ba_state_errors %lu ", ToUlong(aCounters.mEpskcInvalidBaStateErrors));
```

Comment:
nit: add a blank line in the code when you finish a line in the CLI output

Reply:
Done